docs/source/_sidebar.yaml

```diff
@@ -10,6 +10,8 @@ items:
     href: "."
   - label: "Quickstart"
     href: "./quickstart"
+  - label: "Testing & Deployment Workflow"
+    href: "./testing-deployment-workflow"
```

Comment:
@kbychu I really like this addition to the document, but I'm not sure it should be the 3rd item in the sidebar and the label looks a bit too verbose compare to existing ones. Maybe it would be better to move it under the deployment section?

@mabuyo Any thoughts?

<img width="530" height="1068" alt="Image" src="https://github.com/user-attachments/assets/6b7542c4-d419-45ec-8622-63dae7060bd1" />

Reply:
I agree it shouldn't be at the top! I think it should actually be under Guides. The contents even start with `This guide outlines...` It definitely feels more like a conceptual guide to me. 

I also want to rename it: "Development to Production Workflows". It's less about deployment specifically and more talking about how you work with the MCP server in different environments.

@kbychu any concerns here? I can make the changes as I do a more thorough review.

Reply:
@mabuyo Sounds good to me! No concerns.